src/Employer/Employer.Web/Startup.Configure.cs

```diff
@@ -43,17 +44,50 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, IOptions
 
                 app.UseHsts(hsts => hsts.MaxAge(365));
             }
-            
+
+            if (env.IsDevelopment())
+            {
+                IdentityModelEventSource.ShowPII = true;
+            }
             // Add Content Security Policy
+            app.Use(async (context, next) =>
```

Comment:
Can all the Zendesk config be put into it's own Use extension?  e.g. app.UseZenDesk ?